python/tests/expressions/test_literals.py

```diff
@@ -89,12 +89,19 @@ def test_literal_from_none_error() -> None:
         BinaryLiteral,
     ],
 )
-def test_string_literal_with_none_value_error(literal_class: Type[PrimitiveType]) -> None:
+def test_literal_classes_with_none_type_error(literal_class: Type[PrimitiveType]) -> None:
     with pytest.raises(TypeError) as e:
         literal_class(None)
     assert "Invalid literal value: None" in str(e.value)
 
 
+@pytest.mark.parametrize("literal_class", [FloatLiteral, DoubleLiteral])
```

Comment:
Could you add one more tests where we use `literal(float("NaN"))`? The `literal` is meant as the public API, and people shouldn't necessarily initialize the specific literal instances themselves.

Reply:
Done!